bucket/juce.json

```diff
@@ -0,0 +1,24 @@
+{
+    "homepage": "https://juce.com/",
+    "description": "JUCE is an open-source cross-platform C++ application framework. The leading framework for multi-platform audio applications",
+    "version": "5.4.7",
```

Comment:
reorder it by

1. version
2. description
3. homepage

also maybe shorten the description to something along the lines of:

> open-source cross-platform C++ application and audio applications framework.